packages/jest-circus-allure-reporter/src/types/interface.ts

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable require-jsdoc */
```

Comment:
In the interest of getting this code reviewed sooner I have disabled the `require-jsdoc` rule for this file.